components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/store/UserRecoveryDataStore.java

```diff
@@ -37,6 +38,21 @@ UserRecoveryData load(String code) throws
     UserRecoveryData load(User user) throws
             IdentityRecoveryException;
 
+    /**
+     * Load UserRecoveryData from the
+     *
+     * @param code                 Confirmation code
```

Comment:
Shall we end the description with a fullstop?

Reply:
+1. Addressed.